clang/lib/Driver/ToolChains/Gnu.cpp

```diff
@@ -714,6 +714,7 @@ void tools::gnutools::Linker::ConstructJob(Compilation &C, const JobAction &JA,
       if (Args.hasArg(options::OPT_fsycl) &&
           !Args.hasArg(options::OPT_nolibsycl)) {
         CmdArgs.push_back("-lsycl");
+        CmdArgs.push_back("-lsycl-devicelib-host");
```

Comment:
Hi, @mdtoguchi 
libsycl-devicelib-host.a contains implementation for imf device library for SYCL host device, it is linked when user is using sycl compiler.

Thanks very much.